R/utils-hub_tasks_config.R

```diff
@@ -19,3 +19,24 @@ filter_task_groups_to_target <- function(task_groups, target_id) {
 
   return(task_groups)
 }
+
+
+#' Get a character vector of all output types across all task groups
+#' @noRd
+get_output_types <- function(task_groups) {
+  output_types <- purrr::map(
+    task_groups,
+    function(task_group) names(task_group[["output_type"]])
+  ) |>
+    unlist() |>
+    unique()
+  return(output_types)
+}
+
+
+#' Get a boolean indicating whether the target is ordinal
+#' @noRd
+get_target_is_ordinal <- function(task_groups_w_target) {
```

Comment:
**suggestion**

For boolean functions, I would drop the `get_`. When I write boolean functions, I usually try to put them in context of an `if()` statement so that it works in syntax:

```r
if (get_target_is_ordinal()) {
  # what am I doing
}
if (target_is_ordinal()) {
  # do some ordinal target work
}
```

That being said, because you are using this to produce a variable called `target_is_ordinal`, then I would suggest to use something like this:

```suggestion
is_target_ordinal <- function(task_groups_w_target) {
```

this way, you can use:

```r
target_is_ordinal <- is_target_ordinal(task_groups_with_target)
# ...
get_standard_metrics(output_type, target_is_ordinal)
```